kubernetes/README.md

```diff
@@ -3,8 +3,8 @@ No description provided (generated by Swagger Codegen https://github.com/swagger
 
 This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
-- API version: v1.11.3
-- Package version: 7.0.0-snapshot
+- API version: v1.12.1
```

Comment:
k8s latest release is v1.12.0, why is here v1.12.1?

Reply:
Yes I also noticed this during client generation. The API version comes from the currently checked-in openapi spec on k/k release branch https://github.com/kubernetes/kubernetes/blob/release-1.12/api/openapi-spec/swagger.json

```
  "swagger": "2.0",
  "info": {
   "title": "Kubernetes",
   "version": "v1.12.1"
  },
```

The other release branches have the same behavior. I suppose when a patch release is cut (e.g. 1.11.3), the generated openapi spec shows the version for next patch release (1.11.4) that is open for cherry-pick.

I think it's a problem in our client generation code. We should probably curl the [released source code](https://github.com/kubernetes/kubernetes/releases/tag/v1.12.0) to get the openapi spec, instead of downloading from the release branch: https://github.com/kubernetes-client/gen/blob/4aa859fc518385886434d9169a8e383dd1fd16c2/openapi/preprocess_spec.py#L312-L315

We are also exposed to non-deterministic client generation-- currently the client is said to be generated against API version v1.12.1, but there might be new API change introduced later to 1.12.1 that we didn't catch. That being said, in general I don't expect major API change getting introduced to patch release.